ui/webui/src/components/review/ReviewConfiguration.jsx

```diff
@@ -35,15 +35,14 @@ import {
     getDeviceData,
     getAppliedPartitioning,
     getPartitioningRequest,
-    getInitializationMode,
```

Comment:
Maybe we could also remove getInitializationMode function (https://github.com/rhinstaller/anaconda/blob/7779a2812059d42a7554e80f5b3e9ca9a785f9a3/ui/webui/src/apis/storage.js#L291) which is no more used at this point.